app/controllers/bulkrax/importers_controller.rb

```diff
@@ -52,8 +52,16 @@ def create
       field_mapping_params
       if @importer.save
         files_for_import(file, cloud_files)
-        Bulkrax::ImporterJob.perform_later(@importer.id) if params[:commit] == 'Create and Import'
-        redirect_to importers_path, notice: 'Importer was successfully created.'
+        if params[:commit] == 'Create and Import'
+          Bulkrax::ImporterJob.perform_later(@importer.id)
+          redirect_to importers_path, notice: 'Importer was successfully created.'
+        elsif params[:commit] == 'Create and Validate'
+          validate_only = true
```

Comment:
we need to set `@importer.validate_only = true` before the save action, but only if `params[:commit] == 'Create and Validate'`

so move line 59 to before 53, and make it

`@importer.validate_only = true if params[:commit] == 'Create and Validate'`

I would expect ` redirect_to importers_path` to work